TidepoolOnboardingKitUI/Assets.xcassets/Introduction_1_Top.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Introduction_1_Top.pdf",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
```

Comment:
[nit] Should this just be `x1`? Similar comments elsewhere. I'm just not sure if this is the common formatting.

Reply:
I did not create or modify this file. This is what results from simply dragging a PDF from the Finder into the `Asset.xcassets`.

Reply:
It has this format if you have "Individual Scales" selected for this asset in Xcode. For PDFs, generally we'd have a single scale, unless you wanted to have different files used at different scales.

Reply:
Ah, okay. I'll fix all of the pdfs with the next onboarding section. I have the onboarding repo simplification commit stacked on top of this one and don't want to mess that up.